### ISSUE TITLE
tree view overflow hidden when max-width is set

### ISSUE THREAD

**Issue by mrleblanc101** [lexcast/tree-view-autoresize#22]
I had to add this to my stylesheet to make the plugin usable when a max-width is set: 
```css
atom-dock.left .tree-view,
atom-dock.right .tree-view {
    overflow-x: scroll !important;
}
```

**Comment by lexcast** [comment]
Hi @mrleblanc101 
Thanks for reporting, the plugin is unusable with the max-width setting.
Feel free to make a PR

**Comment by lexcast** [comment]
Fixed on #23
